### PR TITLE
[user/configure-device.rst] Made it more beginner-friendly! #62

### LINKED ISSUE #62: [docs] Some possible improvements
So I had a task where I had to a vm with OpenWRT and connect it to OpenWISP locally. There were some challenges that I faced and here's a list of what can be improved so that other students/people don't get confused and know exactly what to do.

1.) I couldn't get the network to work in my OpenWRT vm. 
refer to: https://openwrt.org/docs/guide-user/virtualization/virtualbox-vm
For the second adapter, it wasn't mention what to use. I was using the PCInet-III or something instead of the Intel Pro (82540EM)
I thought it would be good to let you know so that future individuals don't face this problem.

2.) Oh and for the allocation of <ipaddr>, it would be better that on this tutorial if it was set to 192.168.56.3 (more than 192.168.56.2 as it is used for the OpenWISP Server).

3.) After setting up everything, I found out that no devices were being created on the local OpenWISP website.
I had the message "openwisp: OpenWISP config agent started" but there was nothing on the website.
I later found out (thanks to reza) how to fix all that. Some info on: http://openwisp.io/docs/user/configure-device.html need to be improved as there is some confusion. Firstly lines "option uuid" and "option key" should be commented I think, secondly it would be great if it was mentionned that if we had our OpenWISP server locally (e.g 192.168.56.2), then the "option url" would be 'https://192.168.56.2'. That is more beginner-friendly as they know what to write, if <https> should be mentioned and if whether ".../admin" should be written or not.

Otherwise everything is great! I am not saying that anything was bad. On the contrary, the tutorial is really awesome! I am just suggesting how it would better, that's all!

### PR DESCRIPTION
**Pull request by EdgeKing810**
So I added some lines so that people, either students or in general do not face the same issues that I did and save time! Also, please do not remove the GCI task as I have not claimed it yet.

Closes #62

### CODE REVIEW

File: user/configure-device.rst [hunk 1]
```diff
@@ -50,7 +50,11 @@ the terminal.
 
     Continue with the procedures in
     `the OpenWRT virtualbox guide <https://openwrt.org/docs/guide-user/virtualization/virtualbox-vm>`_.
-    It's required to enable SSH access and connect the VM to the internet.
+    It's required to enable SSH access and connect the VM to the internet. Note that both Adapter1 and Adapter2 
+    use "Adapter Type: Intel PRO/1000 MT Desktop". Also, please do not use the same IP Address that you used for 
+    the local OpenWISP website you hosted before. That suggested change applies only when you boot into the OpenWRT 
+    device as per the description of the above link. (For example, if you set 192.168.56.2 as the IP Address of your
```
Comment (nemesisdesign) [inline user/configure-device.rst]: move the final dot after `link` to after the closing parentheses, also change `For` to `for`

File: user/configure-device.rst [hunk 2]
```diff
@@ -76,7 +80,8 @@ Once it's installed, we need to configure it to connect to our OpenWISP2
 controller. To do that, edit the config file located at
 ``/etc/config/openwisp``.
 
-You will see the default config file, something like the following:
+You will see the default config file, something like the following: 
+(If your instance lacks some of the lines in the following, please add them)
```
Comment (nemesisdesign) [inline user/configure-device.rst]: same here, no capital `I` in `if` and move colon to after the closing parentheses, eg:
```
following (if your instance lacks some of the lines in the following, please add them):
```

File: user/configure-device.rst [hunk 3]
```diff
@@ -93,8 +98,8 @@ You will see the default config file, something like the following:
         #option merge_config '1'
         #option test_config '1'
         #option test_script '/usr/sbin/mytest'
-        option uuid ''
-        option key ''
+        #option uuid ''
+        #option key ''
```
Comment (nemesisdesign) [inline user/configure-device.rst]: :+1:

File: user/configure-device.rst [hunk 4]
```diff
@@ -106,7 +111,9 @@ You will see the default config file, something like the following:
 
 Uncomment and change the following fields:
 
-- ``url``: the hostname of your OpenWISP2 controller
+- ``url``: the hostname of your OpenWISP2 controller (For example, if you are hosting your OpenWISP
+  server locally and you set the IP Address to "192.168.56.2", the url would be "https://192.168.56.2/".
+  Note that the "/admin" after the IP Address does not need to be added in the url.)
```
Comment (nemesisdesign) [inline user/configure-device.rst]: this line can be misleading and it's unnecessary, I think you should remove it.

File: user/configure-device.rst [hunk 5]
```diff
@@ -50,7 +50,11 @@ the terminal.
 
     Continue with the procedures in
     `the OpenWRT virtualbox guide <https://openwrt.org/docs/guide-user/virtualization/virtualbox-vm>`_.
-    It's required to enable SSH access and connect the VM to the internet.
+    It's required to enable SSH access and connect the VM to the internet. Note that both Adapter1 and Adapter2
```
Comment (nemesisdesign) [inline user/configure-device.rst]: I'd leave a blank line after `internet.`, eg:

```
... internet.

Note that...
```

File: user/configure-device.rst (hunk 4 above)
Comment (EdgeKing810) [reply user/configure-device.rst]: I didn't remove the line but rather removed the last part of it to make it less confusing.

**Comment by EdgeKing810** [comment]
Applied the changes. Also, it's this GCI task: https://codein.withgoogle.com/tasks/4918992079683584/?sp-organization=4514829432782848&sp-is_beginner=False.

I'm waiting for you to reply on gitter as I need a little help with my previous task before taking this one.

**Comment by nemesisdesign** [comment]
:+1:

**Pull request merged**